app/src/main/java/net/sourceforge/opencamera/cameracontroller/CameraController2.java

```diff
@@ -7310,11 +7318,11 @@ public void initVideoRecorderPostPrepare(
             previewIsVideoMode = true;
             previewBuilder.set(CaptureRequest.CONTROL_CAPTURE_INTENT, CaptureRequest.CONTROL_CAPTURE_INTENT_VIDEO_RECORD);
             camera_settings.setupBuilder(previewBuilder, false);
-            if(video_recorder != null && supports_photo_video_recording && want_video_imu_recording) {
-                // Requested synchronized video and IMU recording
+            if(video_recorder != null && supports_photo_video_recording && (want_video_imu_recording || want_rec_sync)) {
```

Comment:
For now using `(want_video_imu_recording || want_rec_sync)` is fine, but if at some point we have more conditions which lead to the need for frame processing, we would need to change conditions in all these lines again

What I propose is to join these conditions in some single method somewhere and call it (easier to change later)

Reply:
Of course these conditions sometimes need to be checked separately too, so maybe my suggestion is only for cases where we don't care which one is `True`